requirements.txt

```diff
@@ -1,6 +1,7 @@
 # -*- conf-mode -*-
 setuptools>=51.1.0    # Require this first, to prevent later errors
 #
+aiosmtpd>=1.4.6
```

Comment:
IMO moving forward, we should try to separate test only dependencies. Not saying we should start with this PR, but better to start somewhere?

Reply:
This will help us to have clear SBOM for production server.

Reply:
I like the idea, though have some concerns about how we would validate changes to the production dependency list prior to deployment. 

I don't think it's practical to start in this PR because there will be quite a few tooling and deployment adjustments needed regardless of how we do it. It'll need to be its own project.